app/views/browse/note.html.erb

```diff
@@ -71,6 +71,21 @@
   <% end %>
 
   <% if current_user && current_user != @note.author %>
-    <p><small class="text-muted"><%= t "javascripts.notes.show.report_link_html", :link => report_link(t(".report"), @note) %></small></p>
+    <p>
+      <small class="text-muted">
+        <%= t "javascripts.notes.show.report_link_html", :link => report_link(t(".report"), @note) %>
+        <% if @note.status == "open" %>
+          <%= t "javascripts.notes.show.other_problems_resolve", :link => report_link(t(".report"), @note) %>
+        <% elsif @note.status == "closed" %>
+          <%= t "javascripts.notes.show.other_problems_resolved" %>
+        <% end %>
+      </small>
+    </p>
+  <% end %>
+
+  <% if @note.freshly_closed? %>
+    <small class="text-muted">
+      <%= t("javascripts.notes.show.disappear_date", :disappear_in => disappear_in(@note)).html_safe %>
```

Comment:
Avoid using `html_safe`, since that opens up potential for security issues (it just marks the string as html safe, it doesn't do any sanitisation). 

Instead, use a translation key that ends in `_html`. For similar examples, see https://github.com/openstreetmap/openstreetmap-website/pull/2898

Reply:
Ah yes. I had a feeling there might be a way to avoid html_safe. That's better. So it's called `disappear_date_html` in en.yml now.